### ISSUE TITLE
Update-VSTeamRelease fails

### ISSUE THREAD

**Issue by eulesv** [DarqueWarrior/vsteam#221]
Steps to reproduce
------------------

```powershell
Import-Module VSTeam
Set-VSTeamAccount -Account account.visualstudio.com -PersonalAccessToken "token"
$r = Get-VSTeamRelease -ProjectName "project" -Id 1 -Raw
$r.variables.test.value = "test"
Update-VSTeamRelease -ProjectName "project" -Id 1 -Release $r

```

Expected behavior
-----------------

```none
Release 1 should update the value for variable test
```

Actual behavior
---------------

```none
WARNING: VS402865: An empty body in the POST request is not allowed.
Invoke-RestMethod : {"$id":"1","innerException":null,"message":"VS402865: An empty body in the POST request is not 
allowed.","typeName":"Microsoft.VisualStudio.Services.ReleaseManagement.Data.Exceptions.InvalidRequestException, 
Microsoft.VisualStudio.Services.ReleaseManagement2.Data","typeKey":"InvalidRequestException","errorCode":0,"eventId":3000}
At C:\Program Files\WindowsPowerShell\Modules\VSTeam\6.4.1\vsteam.functions.ps1:696 char:15
+       $resp = Invoke-RestMethod @params
+               ~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (System.Net.HttpWebRequest:HttpWebRequest) [Invoke-RestMethod], WebException
    + FullyQualifiedErrorId : WebCmdletWebResponseException,Microsoft.PowerShell.Commands.InvokeRestMethodCommand
```

Environment data
----------------

OS

- [ ] macOS
- [x] Windows
- [ ] Linux

Server

- [ ] TFS 2017
- [ ] TFS 2018
- [ ] Azure DevOps Server
- [x] Azure DevOps Service

```powershell
> Get-VSTeamAPIVersion
Name                           Value
----                           -----
Release                        3.0-preview
Version                        TFS2017
MemberEntitlementManagement
TaskGroups                     3.2-preview.1
DistributedTask                3.0-preview
Core                           3.0
Packaging
ServiceFabricEndpoint
VariableGroups                 3.2-preview.1
Build                          3.0
Git                            3.0
ExtensionsManagement           3.0-preview
Graph
Tfvc                           3.0
```

```powershell
> $PSVersionTable
Name                           Value
----                           -----
PSVersion                      5.1.18362.145
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.18362.145
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

**Comment by DarqueWarrior** [comment]
If you are connecting to Azure DevOps Services you API Versions are wrong. Try calling. 

```PowerShell
Set-VSTeamAPIVersion -Target AzD
```

The versions should be:

```PowerShell
Name                           Value
----                           -----
Release                        5.1-preview
Version                        AzD
MemberEntitlementManagement    5.1-preview
TaskGroups                     5.1-preview.1
DistributedTask                5.0-preview
Core                           5.0
Packaging                      5.1-preview
ServiceFabricEndpoint          5.0-preview
VariableGroups                 5.0-preview.1
Build                          5.0
Git                            5.1-preview
ExtensionsManagement           5.1-preview
Graph                          5.1-preview
Tfvc                           5.0
```

**Comment by eulesv** [comment]
Hi! 

Just tried setting the target version, same error :-(

WARNING: VS402865: An empty body in the POST request is not allowed.

[ Although I learned that if I use the form 'https://dev.azure.com/account' the module switches the API Version to VSTS automatically :-) ]

**Comment by DarqueWarrior** [comment]
I will fine time to do some more testing. Thanks for additional information.

**Comment by pelikanmartin** [comment]
Hi, I have same issue as Roberto (same exception). After some googling the root cause may be special character in my last name "á". Roberto's lastname also contains special character.

Your _apiCall method uses hardcoded Content-Type: application/json. No charset defined.

https://developercommunity.visualstudio.com/content/problem/617371/unable-to-update-release-pipeline-through-rest-api.html

**Comment by pelikanmartin** [comment]
Hi Donovan,

we modified your code to add charset into the API call, import it locally and the problem is gone.

https://github.com/DarqueWarrior/vsteam/blob/master/Source/Public/Update-VSTeamRelease.ps1#L26

changing:
-ContentType 'application/json'
to:
-ContentType 'application/json;charset=utf-8'

It would be great to make such change for every method POST/PUT method.

**Comment by MichelZ** [comment]
Duplicate of #234 
Fixed by #239
